src/schemas/RowSchema.tsx

```diff
@@ -0,0 +1,28 @@
+import { z } from "zod";
+import {CellType, CommentType, Review_Stages} from '../components/TimeCardPage/types'; 
+
+const optionalNumber = z.union([z.undefined(), z.number()]) 
+
+const optionalMember = z.union([z.undefined(), z.object({
+    Start: optionalNumber, End: optionalNumber, Author: optionalNumber
+})]); 
+
+export const CommentSchema = z.object({
+    Author:z.string(), 
+    Type: z.enum([CommentType.Comment, CommentType.Report]), 
+    Timestamp: z.number(), 
+    Content: z.string(), 
+}); 
+
+export type CommentSchema = z.infer<typeof CommentSchema> 
+
+export const RowSchema = z.object({
+    Type: z.enum([CellType.Regular, CellType.PTO]), 
```

Comment:
I don't have this added into backend yet, but can do that and update that in the schema diagram on lucidchart